features/ProductNews.feature

```diff
@@ -0,0 +1,61 @@
+@owner:derek
+@author:lilla
+
+Feature: Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario Outline: Product news updates can be filtered by type
+        Given a user selects from the type dropdown
+        When they select <postType>
+        Then they see a list of <filteredNews>
+        Examples:
+
+            | postType         | filteredNews     |
```

Comment:
Variables in examples can be used in more than one step. In cases like the one you have here, you can simplify to only use one variable